Cargo.toml

```diff
@@ -1,11 +1,8 @@
 [package]
+authors = ["Jake Goulding <jake.goulding@integer32.com>", "Carol (Nichols || Goulding) <carol.nichols@integer32.com>"]
+license = "MIT/Apache-2.0"
```

Comment:
This is the proposed structure when using `cargo add`, so I assumed it was the recommended structure. I can undo this change if you like.